apps/evm/src/libs/tokens/infos/commonTokens/bscTestnet.ts

```diff
@@ -376,4 +377,11 @@ export const tokens: Token[] = [
     symbol: 'SOL',
     asset: solLogo,
   },
+  {
+    address: '0x14AECeEc177085fd09EA07348B4E1F7Fcc030fA1',
+    decimals: 18,
+    symbol: 'PT-clisBNB-APR25',
+    asset: ptClisBNBLogo,
+  },
+  // TODO: add asBNB record once it's been deployed
```

Comment:
`asBNB` on testnet: `0xc625f060ad25f4A6c2d9eBF30C133dB61B7AF072`. Reference: https://github.com/VenusProtocol/oracle/pull/275

Reply:
Thank you, it's added